common/helpers.py

```diff
@@ -66,6 +66,11 @@ def find(iterable, pred):
             return x
 
 
+def pluck(obj, *keys):
```

Comment:
Ah, I meant to call it pick. cf. http://underscorejs.org/#pick

Will fix. 

Might be worth bringing in funcy at some point?